_posts/2020-06-21-one-stop-guide-to-lombok.md

```diff
@@ -0,0 +1,337 @@
+---
+title: "One-stop Guide to Lombok"
+date: 2020-06-21 10:00 +0500
+modified: 2020-06-21 10:15 +0500
```

Comment:
Please add the other fields, too (just copy from the another article).

Also, please add yourself to `authors.yml` (ideally with a picture) and add yourself as an author here in the header.